app/models/user.rb

```diff
@@ -3,4 +3,8 @@ class User < ApplicationRecord
   has_many :suppliers, through: :memberships
   validates :email, presence: true, uniqueness: true
   validates :auth_id, presence: true, uniqueness: true
+
+  def multiple_suppliers
```

Comment:
Should this be a query method - i.e. `multiple_suppliers?`

Reply:
is the convention that if it returns true/false it has the question mark?

Reply:
@mec Yep, it's one of the nice things Ruby does!